op-e2e/system_fpp_test.go

```diff
@@ -27,6 +27,119 @@ func TestVerifyL2OutputRootDetached(t *testing.T) {
 	testVerifyL2OutputRoot(t, true)
 }
 
+func TestVerifyL2OutputRootEmptyBlock(t *testing.T) {
+	testVerifyL2OutputRootEmptyBlock(t, false)
+}
+
+func TestVerifyL2OutputRootEmptyBlockDetached(t *testing.T) {
+	testVerifyL2OutputRootEmptyBlock(t, true)
```

Comment:
This flaked: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/17578/workflows/48c306ad-b1af-4334-8bbc-a88a4eeeb2a0/jobs/676022